app/routes/bdb/utils.js

```diff
@@ -64,8 +64,11 @@ export const selectMostProminentStatus = (
 
 export const semesterNameOf = (index: number) => {
   const indexToSemesterName = {
-    '0': 'spring',
-    '1': 'autumn',
+    // Non-string literal property keys is not supported in flow yet
+    // $FlowFixMe
+    0: 'spring',
```

Comment:
Why are you changing this? Objects can only have strings as keys, so this doesn't change anything?

Reply:
eslint were changing these by rule. Not an issue in the latest version.